src/MetricGraphScene.tsx

```diff
@@ -160,3 +178,14 @@ function getStyles(theme: GrafanaTheme2, headerHeight: number, trail: DataTrail)
     }),
   };
 }
+
+function updateActionBarHeight() {
```

Comment:
i think this function should also be updated to use the ref instead of querySelector.

So something like:

const actionBar = controlsContainer.current;